### ISSUE TITLE
Dadgame: Invisible Tutorial Text and mobile graphical issues.

### ISSUE THREAD

**Issue by waspennator** [ruffle-rs/ruffle#8945]
### Describe the bug

https://www.newgrounds.com/portal/view/510279
Game itself seems to be reasonably playable, but when first starting up a game by choosing any difficulty on my s22, the screen that asks if you want to play the tutorial won't show the text correctly in the middle.

![Screenshot_20221227_171225_Chrome](https://user-images.githubusercontent.com/78768499/209741030-46b474dd-4e83-4e5e-9b31-70c2df61bc09.jpg)

The game seems to suffer from similar graphical issues that #8917 does with the usual
"ruffle_web-wasm_extensions.js:1094 WebGL: INVALID_VALUE: texImage2D: width or height out of range"
error
![Screenshot_20221227_171234_Chrome](https://user-images.githubusercontent.com/78768499/209741103-3e0391ca-b601-483b-abb2-8266a9151327.jpg)

### Expected behavior

The text for the screen asking if you want to do the tutorial should be showing like this.
![Dad2](https://user-images.githubusercontent.com/78768499/209741572-67d7e910-e13a-4a0c-8ad6-6fccbd6b326f.png)

The ingame background should look like this.
![Dad](https://user-images.githubusercontent.com/78768499/209741598-579800ad-32da-4fef-9c83-76cf555c89a2.png)

### Affected platform

Browser's extension

### Operating system

Windows 11 and Android 13

### Browser

Edge 108

**Comment by Lord-McSweeney** [comment]
I can see the "PLAY TUTORIAL?" text on the web version, on Chrome (at least, when selecting Easy difficulty).

**Comment by waspennator** [comment]
Retesting it again on my end, seems like the play tutorial text does show up fine on the web version, still not showing up at all on the mobile version.

**Comment by n0samu** [comment]
Yeah I'm not having any issue with the Play Tutorial text either.

**Comment by waspennator** [comment]
> Yeah I'm not having any issue with the Play Tutorial text either.

While it seems to be fine for me on web now, mobile is still acting weird with it. Most times the text won't be there, sometimes if I click around in the level select, the text will show up again for a few times, then the next time I click on a level or difficulty, it'll vanish.
